readthedocs/config/tests/test_yaml_loader.py

```diff
@@ -12,9 +12,13 @@ def test_yaml_load_safely():
     content = textwrap.dedent('''
     int: 3
     float: !!float 3
-    unknown: !!python/name:unknown_funtion
+    name: !!python/name:python_function
+    unknown: !!python/module:python_module
     ''')
     data = yaml_load_safely(content)
+    name = data.pop('name')
     assert data == expected
```

Comment:
We shouldn't `.pop` the item here and update the `expected` instead.

Reply:
This is what I did originally, but then the object in `expected` must be an instance of `ProxyPythonName`, and when comparing it with the loaded one, it's false, because they are not the same object. This could be fixed by adding an `__eq__` method on the `ProxyPythonName` class, but I felt like it was adding useless code just for the test to pass. However if you think this is the way to go, I'll happily add this `__eq__` method :slightly_smiling_face: Maybe there is another solution though?

Reply:
I think we should move the assert to be just after the pop

Reply:
Done. Reorganized the assert statements and added a comment to explain why we pop.